src/app/main/component/events/components/images-container/images-container.component.ts

```diff
@@ -65,6 +67,12 @@ export class ImagesContainerComponent implements OnInit {
     }
   }
 
+  ngOnChanges() {
```

Comment:
6350

Reply:
what does it mean here and elsewhere?

Reply:
It was for notes for the demo meeting. I can delete these notes since they are not necessary